cloud/types.go

```diff
@@ -220,15 +223,20 @@ type (
 
 	// DeploymentReviewRequest is the review_request object.
 	DeploymentReviewRequest struct {
-		Platform    string    `json:"platform"`
-		Repository  string    `json:"repository"`
-		CommitSHA   string    `json:"commit_sha"`
-		Number      int       `json:"number"`
-		Title       string    `json:"title"`
-		Description string    `json:"description"`
-		URL         string    `json:"url"`
-		Labels      []Label   `json:"labels,omitempty"`
-		Reviewers   Reviewers `json:"reviewers,omitempty"`
+		Platform              string    `json:"platform"`
+		Repository            string    `json:"repository"`
+		CommitSHA             string    `json:"commit_sha"`
+		Number                int       `json:"number"`
+		Title                 string    `json:"title"`
+		Description           string    `json:"description"`
+		URL                   string    `json:"url"`
+		Labels                []Label   `json:"labels,omitempty"`
+		Reviewers             Reviewers `json:"reviewers,omitempty"`
+		ChangesRequestedCount int       `json:"changes_requested_count"`
+		ApprovedCount         int       `json:"approved_count"`
+		ChecksTotalCount      int       `json:"checks_total_count"`
```

Comment:
Is it the sum of `checks_failure_count` + `checks_success_count`?

Reply:
No, it's not the same. There's at least still "skipped" checks, and maybe others.